build-core-host-pc.sh

```diff
@@ -19,3 +19,6 @@ b_log "Building phoenix-rtos-filesystems"
 
 b_log "Building phoenix-rtos-devices"
 (cd phoenix-rtos-devices && make $MAKEFLAGS $CLEAN all)
+
+b_log "Building phoenix-rtos-tests"
+(cd phoenix-rtos-tests && make $MAKEFLAGS $CLEAN all)
```

Comment:
do we need to install the built tests somewhere? From https://github.com/phoenix-rtos/phoenix-rtos-tests/pull/9 I think they need to be placed in `_boot`? 

Or should I merge this PR right away and the needed changes will follow in separate PR?

Reply:
Is a part of https://github.com/phoenix-rtos/phoenix-rtos-project/pull/63 - sorry for the lack of link.

I handled host's test similar to `ia32-generic` to simplify building:
1. Tests are build in the "core" part
2. Then they are installed in a fake filesystem in the per target routine [b_test_target](https://github.com/phoenix-rtos/phoenix-rtos-project/pull/63/files#diff-7d67bef63d3624f3ba5a5c70b477f3cb2033a61c6476d13f6b1c470005ee0a91R19)
3. Then copied from filesystem to prog.stripped

Now I think it may be a little bit overcomplicated and maybe we can just copy tests from prog.stripped.

Reply:
I've put another comment in phoenix-rtos-project, but the proposed approach there is also far-fetched (currently the we would need to unpack rootfs in github action so there is no real gain. In the future I assume there will be "upload to device" step in the runner - which for host-pc may be implemented as unpacking rootfs.tar). The current approach is enough for now :).

LGTM.